Detectors/ITSMFT/MFT/workflow/src/TrackerSpec.cxx

```diff
@@ -92,10 +92,10 @@ void TrackerDPL::run(ProcessingContext& pc)
               << mc2rofs.size() << " MC events";
   }
 
-  std::vector<o2::mft::TrackMFTExt> tracks;
+  //std::vector<o2::mft::TrackMFTExt> tracks;
```

Comment:
Also in this device the clusters are extracted (L73,74) via vectors (i.e. copy is created) instead of accessing them via span. 
Since this is out of scope of this PR, I propose that we merge it, but then please open another PR changing vectors to spans.